cmd/podman/images.go

```diff
@@ -124,6 +127,7 @@ func imagesInit(command *cliconfig.ImagesValues) {
 	flags.BoolVar(&command.NoTrunc, "no-trunc", false, "Do not truncate output")
 	flags.BoolVarP(&command.Quiet, "quiet", "q", false, "Display only image IDs")
 	flags.StringVar(&command.Sort, "sort", "created", "Sort by created, id, repository, size, or tag")
+	flags.BoolVarP(&command.History, "history", "", false, "Display the image name history as well")
```

Comment:
I'd drop "as well"

Reply:
If we don't support it for remote, we need to hide flag. Any reason to not support it on remote? I am okay with only supporting it for the native binary as extending the API is currently a lot of work.

Reply:
I added some code parts regarding the API but it looks like that it does not work right now. If I do something like:
```
sudo ./bin/podman varlink --timeout=0 unix:/run/podman/io.podman
```
and then query the API
```
> sudo ./bin/podman-remote images --history
REPOSITORY                 TAG      IMAGE ID       CREATED          SIZE      HISTORY
localhost/test             latest   e5626a0c73d2   32 minutes ago   2.07 kB   <none>
<none>                     <none>   930a10723169   32 minutes ago   2.07 kB   <none>
```
This does not look correct, since the output should be:
```
> sudo ./bin/podman images --history
REPOSITORY                 TAG      IMAGE ID       CREATED          SIZE      HISTORY
localhost/test             latest   e5626a0c73d2   32 minutes ago   2.07 kB   <none>
<none>                     <none>   930a10723169   32 minutes ago   2.07 kB   localhost/test:latest
```

Is there any location I missed updating?